cesm/driver/esm_time_mod.F90

```diff
@@ -54,7 +54,7 @@ module esm_time_mod
 !===============================================================================
 
   subroutine esm_time_clockInit(ensemble_driver, instance_driver, logunit, maintask, rc)
-
+    use nuopc_shr_methods, only : get_minimum_timestep, dtime_drv
```

Comment:
Comment that nuopc_shr_methods is now in the cdeps code and cdeps is always built alongside of cmeps - so this is available to cmeps.